packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -392,97 +392,61 @@ export class PluginVscodeCommandsContribution implements CommandContribution {
                 id: 'vscode.executeDefinitionProvider'
             },
             {
-                execute: ((resource: URI, position: Position) => {
-                    const args = {
-                        resource: monaco.Uri.from(resource),
-                        position: position
-                    };
-                    return commands.executeCommand<Location[]>('_executeDefinitionProvider', args);
-                })
+                execute: ((resource: URI, position: Position) =>
```

Comment:
Are there new commands for semantic ranges?

Reply:
I didn't find any commands related to `DocumentSemanticTokensProvider ` [here](https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/workbench/api/common/extHostApiCommands.ts)